src/fparser/readfortran.py

```diff
@@ -140,15 +141,16 @@
 import re
 import os
 import sys
+import six
 import tempfile
 import traceback
 import logging
-from cStringIO import StringIO
+from io import StringIO
```

Comment:
I may be wrong but I don't think this import is required since the code now uses six.StringIO instead.